### PR TITLE
Typos fixed.

### PR DESCRIPTION
**Pull request by TowleT**
Typos fixed.

### CODE REVIEW

**Comment by jeffheifetz** [comment]
Please remove the *.orig files

File: api/blackberry_io_dir.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ blackberry.io.dir = {
 	
 	/**
 	* Rename a given directory. 
-	* @param {String} path path location of the directory to be renamed.
+	* @param {String} ath Path location of the directory to be renamed.
```
Comment (jeffheifetz) [inline api/blackberry_io_dir.js:80]: This should be 

```
@param {String} path Path location of the directory to be renamed.
```

**Comment by jeffheifetz** [comment]
The phrase "path location" is used a lot in the blackberry_io_dir file. This sounds unnecessary to me, perhaps just path or location would suffice.

**Comment by jeffheifetz** [comment]
Please read issue #150 as it provides some input about your changes for DeviceMotion

**Comment by TowleT** [comment]
Made changes to deviceMotion

**Pull request closed without merging**